builtin/k8s/releaser.go

```diff
@@ -91,34 +92,60 @@ func (r *Releaser) Release(
 		labelId: target.Id,
 	}
 
+	if (r.config.Port != 0 || r.config.NodePort != 0) && r.config.Ports != nil {
+		return nil, fmt.Errorf("Cannot define both 'Ports' and 'port' or 'node_port'." +
```

Comment:
nit: is the capitalization supposed to differ across these terms?

Reply:
Ah, nope! Ports is how it's referred to internally, but in the config it's lower case. So it should be lower case here too.